array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map((num) => num * 2);
+console.log('All numbers doubled:', doubled);
+
+const prices = numbers.map((num) => `$${num}.00`);
```

Comment:
This only works because the numbers are integers. The general solution is to use `.toFixed(2)`.